ext/bcmath/libbcmath/src/init.c

```diff
@@ -82,6 +82,20 @@ void bc_init_numbers(void)
 	BCG(_two_)->n_value[0] = 2;
 }
 
+static void _bc_force_free_number(bc_num *num)
```

Comment:
Function names starting with underscores should be avoided in new code because in C a name starting with underscore is reserved.